frontend/rust-lib/flowy-database2/src/entities/filter_entities/checklist_filter.rs

```diff
@@ -36,22 +34,10 @@ impl std::convert::TryFrom<u8> for ChecklistFilterConditionPB {
   }
 }
 
-impl FromFilterString for ChecklistFilterPB {
```

Comment:
It helps you fast navigating when using a trait instead of (u8, string). I prefer using trait instead of tuple

For example
<img width="1286" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/86001920/37c760ee-194a-4486-aa58-76be2985a43b">

Reply:
Also it is easy to extend when using trait